intel_extension_for_transformers/neural_chat/tools/rome/examples/README.md

```diff
@@ -0,0 +1,104 @@
+# Prerequisite​
+
+## Environment​
+Recommend python 3.9 or higher version.
+```shell
+pip install -r requirements.txt
+```
+
+# Edit Knowledge of LLMs
+
+This example aims to demonstrate how to use [Rank-One Model Editing (ROME)](https://arxiv.org/pdf/2202.05262.pdf) algorithm to edit the inherent knowledge of LLMs.
```

Comment:
it would be better to add a brief description of the model editing and the influence of the model editing in the introduction part

Reply:
added.